src/net/codec.rs

```diff
@@ -1017,6 +1034,7 @@ impl StacksMessageCodec for StacksMessageID {
             x if x == StacksMessageID::Pong as u8 => StacksMessageID::Pong,
             x if x == StacksMessageID::NatPunchRequest as u8 => StacksMessageID::NatPunchRequest,
             x if x == StacksMessageID::NatPunchReply as u8 => StacksMessageID::NatPunchReply,
+            x if x == StacksMessageID::Attachment as u8 => StacksMessageID::Attachment,
```

Comment:
Same here -- Attachments aren't p2p messages, so there shouldn't be a `StacksMessageID` for it.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2013/commits/52b70da1a59b770c39484522f20698f478b90d1a